cmd/agent/main.go

```diff
@@ -55,11 +55,9 @@ func main() {
 			logrus.Warnf("failed to reconcile kubelet, error: %v", err)
 		}
 
-		logrus.SetOutput(colorable.NewColorableStdout())
+		setLogrus()
```

Comment:
Nit: `configureLogrus` sounds a bit clearer
```suggestion
		configureLogrus()
```